conf.py

```diff
@@ -61,7 +61,7 @@
 pygments_style = 'sphinx'
 
 extlinks = {
-    'manual': ('http://docs.mongodb.com/manual%s', ''),
+    'manual': ('https://docs.mongodb.com/manual%s', ''),
```

Comment:
Ignore this -- I'm reverting this change since it doesn't actually do anything anyway.